.eslintrc.js

```diff
@@ -120,7 +120,7 @@ module.exports = {
             "*********************************************************************",
             {
               "pattern": "^ \\* Copyright \\(c\\) \\d{4}(-\\d{4})* Red Hat, Inc\\.$",
-              "template": " * Copyright (c) 2020 Red Hat, Inc."
+              "template": " * Copyright (c) 2021 Red Hat, Inc."
```

Comment:
@benoitf I'm not sure about this change, does this generate copyright headers? Maybe it should be `2020-2021`

Reply:
this change is perfect.
This template is used to generate headers when you create files and miss the copyright header. (eslint:fix command).
So it'll happen only for new files that will be added in 2021 because all the other files already have valid headers.